i2csoft/i2csoft_atsamd51.go

```diff
@@ -0,0 +1,23 @@
+//go:build atsamd51 || atsame5x
+// +build atsamd51 atsame5x
+
+package i2csoft
+
+import (
+	"device/arm"
+)
+
+func (i2c *I2C) wait() {
+	// atsamd51
+	//  1 : about 388kHz
+	// 17 : about 97kHz
```

Comment:
The setting values for samd51 (120MHz) and samd21 (48MHz) are now almost identical. Is it possible that samd21 is faster? Or is samd51 slower?

Reply:
That was my note for wait.
I have corrected the comment.